guides/common/modules/proc_installing-the-satellite-server-packages.adoc

```diff
@@ -22,13 +22,13 @@ ifdef::foreman-deb[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-update}
+# {client-package-update-el8}
 ----
 . Install `{foreman-installer-package}`:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} {foreman-installer-package}
+# {client-package-install-el8} {foreman-installer-package}
```

Comment:
Why `client-*` on ProjectServer?

Reply:
Mistake; this is foreman-deb only; therefore, we can use "project-package-*". Fixed.